### DISCUSSION TITLE
Logging request object + response

### DISCUSSION THREAD

**Discussion by RedSpid3r** [honojs/hono#1440] [Q&A]
Hi, i am re-writing the API for one of my projects with Hono and I am wondering how I would go about accessing the request and response objects. I want to be able to log things like the response status, duration etc. as well as the request time and IP so I can monitor API health via piping this info into my existing Grafana Loki backend.

I note there is a "Logger" middleware but the docs are extremely vague and don't offer any information on how to use or access it.

Any help would be appreciated. Thanks

**Comment by yusukebe** [comment]
Hi,

Perhaps, the Sentry Middleware will help you:

https://github.com/honojs/middleware/tree/main/packages/sentry
